### ISSUE TITLE
Change Callbacks Syntax Support

### ISSUE THREAD

**Issue by dngulin** [kizeevov/slint-idea-plugin#75]
[Change callbacks](https://docs.slint.dev/latest/docs/slint/guide/language/coding/properties/#change-callbacks) are reported as errors:

![Image](https://github.com/user-attachments/assets/13bc265b-5839-4f9f-84b7-b5847dc256dc)

```
'(', ':', <=>, '=>' or '{' expected, got 'traget-opacity'
```

**Comment by kizeevov** [comment]
Feature added, please check

**Comment by dngulin** [comment]
It is fixed in the last plugin version. Thank you!
